app/views/welcome/first_time.html.erb

```diff
@@ -1,9 +1,9 @@
-<div class="row justify-content-center">
+<main class="row justify-content-center" id="content">
   <div class="card col-md-10 col-lg-8 col-xl-7 col-xxl-6 mt-5 mb-5">
     <div class="card-body">
-      <h5 class="card-title">Welcome!</h5>
+      <h1 class="card-title">Welcome!</h1>
```

Comment:
Select a heading based on semantics, not display (which can be controlled by CSS).